superset/assets/javascripts/explorev2/actions/exploreActions.js

```diff
@@ -94,8 +94,8 @@ export function changeFilter(filter, field, value) {
 }
 
 export const SET_FIELD_VALUE = 'SET_FIELD_VALUE';
-export function setFieldValue(datasource_type, key, value, label) {
-  return { type: SET_FIELD_VALUE, datasource_type, key, value, label };
+export function setFieldValue(datasource_type, key, value) {
```

Comment:
this label is used for setting datasource_name when switching datasource, because we require datasource_name in overwrite_and_save_slice [https://github.com/airbnb/superset/blob/34d0dd9d6e074e41968c39247d66e19fd551c57b/superset/views.py#L1557](url)